templates/sat.html

```diff
@@ -11,8 +11,13 @@ <h1>Sat {{ self.sat.n() }}</h1>
   <dt>offset</dt><dd>{{ self.sat.third() }}</dd>
   <dt>rarity</dt><dd><span class={{self.sat.rarity()}}>{{ self.sat.rarity() }}</span></dd>
   <dt>timestamp</dt><dd><time>{{self.blocktime.timestamp()}}</time>{{self.blocktime.suffix()}}</dd>
-%% if let Some((inscription)) = &self.inscription {
-  <dt>inscription</dt><dd class=thumbnails>{{ Iframe::thumbnail(*inscription) }}</dd>
+%% if !self.inscriptions.is_empty() {
```

Comment:
Did you visually confirm that this looks good?

Reply:
Yes. It's the same layout we use for showing all inscriptions in an output.